src/components/__tests__/__snapshots__/MapPanel.spec.jsx.snap

```diff
@@ -173,9 +173,11 @@ exports[`component:MapPanel renders Print without crashing 1`] = `
     className="row-chart-section"
   >
     <h3>
-      Product by highest complaint volume
+      Sub-products, by product from Invalid date to Invalid date
```

Comment:
could you update setupSnapshot's state.query with date_received_min and date_received_max values so that these don't say invalid date?